### PR TITLE
StructureDefinition-cqf-notDoneReason creation

### PR DESCRIPTION
**Pull request by bryantaustin13**
Part of https://jira.hl7.org/browse/FHIR-45369 Update QI-Core to use extensions pack

for cqf-notDoneReason

### CODE REVIEW

File: input/definitions/StructureDefinition/StructureDefinition-cqf-notDoneReason.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      "uri" : "http://hl7.org/v3",
+      "name" : "RIM Mapping"
+    }
+  ],
+  "kind" : "complex-type",
+  "abstract" : false,
+  "context" : [
+    {
+      "type" : "element",
+      "expression" : "Resource"
```
Comment (brynrhodes) [inline input/definitions/StructureDefinition/StructureDefinition-cqf-notDoneReason.json:50]: For notDoneReason, what resources specifically do we expect this to be usable on? In QICore, this is only in the ObservationCancelled profile, so is Observation the only one? I'd say let's start there

**Review by brynrhodes — CHANGES_REQUESTED** [review]
Please tighten the extension context to Observation

File: input/definitions/StructureDefinition/StructureDefinition-cqf-notDoneReason.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+  "experimental" : false,
+  "date" : "2024-08-20",
+  "publisher" : "HL7 International / FHIR Infrastructure",
+  "contact": [
+    {
+      "name" : "Clinical Quality Information WG",
+      "telecom": [
+        {
+          "system": "url",
+          "value" : "http://www.hl7.org/Special/committees/fiwg"
```
Comment (brynrhodes) [inline input/definitions/StructureDefinition/StructureDefinition-cqf-notDoneReason.json:21]:
```suggestion
          "value" : "http://www.hl7.org/Special/committees/cqi"
```

File: input/definitions/StructureDefinition/StructureDefinition-cqf-notDoneReason.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
+{
```
Comment (brynrhodes) [inline input/definitions/StructureDefinition/StructureDefinition-cqf-notDoneReason.json:1]: Also, please move the definition to the definitions/Resource folder

File: input/definitions/StructureDefinition/StructureDefinition-cqf-notDoneReason.json [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+      ]
+    }
+  ],
+  "description" : "Indicates the reason the event was not done.",
+  "jurisdiction" : [
+    {
+      "coding" : [
+        {
+          "system" : "urn:iso:std:iso:3166",
+          "code" : "US"
```
Comment (brynrhodes) [inline input/definitions/StructureDefinition/StructureDefinition-cqf-notDoneReason.json:32]: Remove this jurisdiction, it's not appropriate in the extensions pack

**Comment by bryantaustin13** [comment]
All change requests completed.

File: input/definitions/Resource/ValueSet-negation-reason-example.json [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+  "meta": {
+    "lastUpdated": "2024-07-01T07:07:38.434+11:00",
+    "profile": [
+      "http://hl7.org/fhir/StructureDefinition/shareablevalueset"
+    ]
+  },
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/StructureDefinition/structuredefinition-wg",
+      "valueCode": "fhir"
```
Comment (brynrhodes) [inline input/definitions/Resource/ValueSet-negation-reason-example.json:14]:
```suggestion
      "valueCode": "cqi"
```

File: input/definitions/Resource/ValueSet-negation-reason-example.json [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+    }
+  ],
+  "name": "NegationReasonExample",
+  "title": "Negation Reason Example",
+  "status": "active",
+  "experimental": false,
+  "date": "2024-07-01",
```
Comment (brynrhodes) [inline input/definitions/Resource/ValueSet-negation-reason-example.json:21]:
```suggestion
  "date": "2024-07-01",
  "publisher" : "HL7 International / Clinical Quality Information",
  "contact": [
    {
      "name" : "Clinical Quality Information WG",
      "telecom": [
        {
          "system": "url",
          "value" : "http://www.hl7.org/Special/committees/cqi"
        }
      ]
    }
  ],
```

**Review by brynrhodes — CHANGES_REQUESTED** [review]
A few more minor tweaks

**Review by brynrhodes — APPROVED** [review]

**Pull request closed without merging**